routes/messages.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const router = express.Router();
+const joi = require("@hapi/joi");
+const Message = require("../models/Message");
+
+const messageValidation = joi.object({
+  fromUserId: joi.string().required(),
+  toUserId: joi.string().required(),
+});
+
+router.get("/", async (req, res) => {
```

Comment:
Axios on the frontend doesn't allow request bodies to be passed in with the get requests (probably why the tutorial guy used a post request). I changed my local copy to use `req.query` with the get request. Not sure if you want me to push it up or if you want to change it yourself

Reply:
Yes could you push it up to this same branch so it shows up here! Thanks